environment/src/main/java/com/sequenceiq/environment/environment/validation/cloudstorage/EnvironmentBackupLocationValidator.java

```diff
@@ -0,0 +1,36 @@
+package com.sequenceiq.environment.environment.validation.cloudstorage;
+
+import java.util.Optional;
+
+import com.sequenceiq.environment.environment.dto.EnvironmentBackup;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.validation.ValidationResult;
+import com.sequenceiq.environment.environment.domain.Environment;
+
+@Component
+public class EnvironmentBackupLocationValidator {
+
+    private final CloudStorageLocationValidator cloudStorageLocationValidator;
+
+    public EnvironmentBackupLocationValidator(CloudStorageLocationValidator cloudStorageLocationValidator) {
+        this.cloudStorageLocationValidator = cloudStorageLocationValidator;
+    }
+
+    /**
+     * Validate telemetry related logging storage location.
+     * Currently, filter out cloudwatch (or any other cloud logging service) related validations
+     */
+    public ValidationResult validateBackupStorageLocation(Environment environment) {
+        ValidationResult.ValidationResultBuilder resultBuilder = new ValidationResult.ValidationResultBuilder();
+        Optional.ofNullable(environment.getBackup())
+                .filter(backup -> isCloudStorageEnabled(backup) && backup.getCloudwatch() == null)
```

Comment:
do I understand correctly if cloud storage location is validated only if there is no cloudwatch field  set on the backup?

Reply:
@oleewere https://github.com/hortonworks/cloudbreak/pull/6851/files

This is a direct copy of @oleewere's change in the above pull request from the EnvironmentLogStorageLocationValidator. _.filter(logging -> logging.getCloudwatch() == null)_. As I am only separating backup bucket from logging bucket, I am coping the same validation policy as what is working currently. 

Oliver specifically commented "* Currently, filter out cloudwatch (or any other cloud logging service) related validations"

Hi @oleewere Would you mind checking why it is filtering by cloudwatch == null in your previous commits? Thanks!

Reply:
sorry for late response, so we can send log data to cloudwatch instead of s3, but the instance profile is validated against s3, so we only do the validation if no cloudwatch enabled. all of this won't be related for backup